dataclients/kubernetes/eastwest.go

```diff
@@ -43,3 +46,52 @@ func createEastWestRouteRG(name, ns, postfix string, r *eskip.Route) *eskip.Rout
 	ewr.Predicates = p
 	return ewr
 }
+
+func applyEastWestRange(eWRange map[string]string, host string, routes []*eskip.Route) {
+	for d, n := range eWRange {
+		if strings.HasSuffix(host, d) {
+			applyEastWestRangePredicates(routes, n)
+		}
+	}
+}
+
+func applyEastWestRangePredicates(routes []*eskip.Route, n string) {
+	for _, route := range routes {
+		route.Predicates = append(route.Predicates, &eskip.Predicate{
+			Name: "SourceFromLast",
```

Comment:
That’s the wrong predicate. We only want to allow https://opensource.zalando.com/skipper/reference/predicates/#clientip without xff headers. Within cluster doesn’t pass the cloud loadbalancer

Reply:
good one. I will update the narrative too.

Reply:
for the record, I'm not sure if by 
> without xff headers

you were referring to check whether the header is present, but in any case, that's not necessary since faking the `Remote-Addr` would result in a failed syn-ack.

Reply:
@jonathanbeber I think sandor is referring to the fact that if you have an NLB (not ALB) in front of skipper then you can fake the xff header if `SourceFromLast` is used. This is why we created `ClientIP` which was done quite recently #1614 :)

Reply:
That's also the reason `SourceFromLast` was mentioned in the internal narrative, because we didn't have anything else at the time and didn't yet consider NLB as an option.

Reply:
Thanks, @mikkeloscar, then, indeed, using just `ClientIP` is considered enough, I'm marking it as resolved.

Reply:
reopening it as after some test we found out that just `ClientIP` won't work for us: https://github.com/zalando/skipper/issues/1526#issuecomment-756903746

Reply:
this means we will also have to support multiple CIDRs per domain (or at least two).

Reply:
Multiple cidr makes sense for migrations, too. Think about migration of subnetwork we already did. to increase the network size for clusters.

Reply:
it's done now, it accepts predicates instead of CIDRs, what gives us a lot more flexibility.